R/env-special.R

```diff
@@ -264,19 +264,23 @@ is_namespace <- function(x) {
   isNamespace(x)
 }
 
-#' Is a package installed in the library?
+#' Are packages installed in any of the libraries?
 #'
-#' This checks that a package is installed with minimal side effects.
-#' If installed, the package will be loaded but not attached.
+#' This checks that packages are installed with minimal side effects.
+#' If installed, the packages will be loaded but not attached.
 #'
-#' @param pkg The name of a package.
-#' @return `TRUE` if the package is installed, `FALSE` otherwise.
+#' @param pkg The package names.
+#' @return A named logical vector of the same length as `pkg`. `TRUE` for each
+#'   installed package and `FALSE` for each package not found in any of the
+#'   libraries.
 #' @export
 #' @examples
-#' is_installed("utils")
+#' is_installed(c("graphics", "utils"))
 #' is_installed("ggplot5")
 is_installed <- function(pkg) {
-  is_true(requireNamespace(pkg, quietly = TRUE))
+  vapply(pkg,
```

Comment:
Can you use `map_lgl()` and follow the local style please?

Reply:
done